src/Symfony/Component/Security/Core/Resources/translations/security.nl.xlf

```diff
@@ -66,6 +66,14 @@
                 <source>Account is locked.</source>
                 <target>Account is geblokkeerd.</target>
             </trans-unit>
+            <trans-unit id="17">
+                <source>Too many failed login attempts, please try again later.</source>
+                <target>Te veel onjuiste inlogpogingen, probeer het later nogmaals.</target>
```

Comment:
on `5.x`, this is translated as `Er waren teveel mislukte inlogpogingen, probeer het later opnieuw.`
Which version is the good one?

Reply:
Please submit again on branch 3.4 if this needs an update.

Reply:
"Er waren teveel" is past tense, either "There were to many". The translation above is the correct one.

Reply:
I agree, "Te veel onjuiste inlogpogingen" is a better translation.

Reply:
Thank you